main.go

```diff
@@ -231,17 +231,45 @@ func getSalaryOfAgencyMonthYear(c echo.Context) error {
 		log.Printf("[salary agency month year] error getting data for second screen(mes:%d ano:%d, orgao:%s):%q", month, year, agencyName, err)
 		return c.JSON(http.StatusBadRequest, fmt.Sprintf("Parâmetro ano=%d, mês=%d ou nome do orgão=%s são inválidos", year, month, agencyName))
 	}
-	var employees []models.Employee
+	members := map[int]int{10000: 0, 20000: 0, 30000: 0, 40000: 0, 50000: 0, -1: 0}
+	servers := map[int]int{10000: 0, 20000: 0, 30000: 0, 40000: 0, 50000: 0, -1: 0}
+	inactive := map[int]int{10000: 0, 20000: 0, 30000: 0, 40000: 0, 50000: 0, -1: 0}
+	maxSalary := 0.0
+
 	for _, employeeAux := range agencyMonthlyInfo.Employee {
-		newEmployee := models.Employee{
-			Name:   employeeAux.Name,
-			Wage:   *employeeAux.Income.Wage,
-			Perks:  employeeAux.Income.Perks.Total,
-			Others: employeeAux.Income.Other.Total,
-			Total:  employeeAux.Income.Total}
-		employees = append(employees, newEmployee)
+		if employeeAux.Income.Total > maxSalary {
+			maxSalary = employeeAux.Income.Total
+		}
+		salary := employeeAux.Income.Total
+		var salaryRange int
+		if salary <= 10000 {
+			salaryRange = 10000
+		} else if salary > 10000 && salary <= 20000 {
```

Comment:
Não precisa do `if salary > 10000`, ele só entra aqui se for maior que 10000 (já é o caso contrário do if acima).

Da mesma forma para todos os abaixo.